runatlantis.io/docs/custom-workflows.md

```diff
@@ -47,6 +47,15 @@ workflows:
       - init
       - plan:
           extra_args: ["-var-file", "production.tfvars"]
+    apply:
+      steps:
+        - apply:
+            extra_args: ["-var-file", "production.tfvars"]
+    import:
+      steps:
+        - init
+        - import:
+            extra_args: ["-var-file", "production.tfvars"]
```

Comment:
There is a section at the bottom of this file called step which documents the init plan and apply steps. We can add import to that too.

https://deploy-preview-2783--runatlantis.netlify.app/docs/custom-workflows.html#step

Reply:
👌 https://github.com/runatlantis/atlantis/pull/2783/commits/79ba39edfb1ce5c775c46dcbb70fe26f48157361